AndriiTsurkov/HW3/2.py

```diff
@@ -0,0 +1,15 @@
+try:
+    string_of_number = int(input("Please enter a four digit natural number: "))
+except:
+    print("You have entered not number.")
+    exit(0)
+
```

Comment:
according to the condition, it was necessary to calculate the product of the digits of the number
prod_of_digits = 0
for i in string_of_number:
    prod_of_digits *= int(i)

Reply:
I understood my mistake. Thank you.